packages/docs-app/src/components/resources.tsx

```diff
@@ -18,11 +18,15 @@ const RESOURCES: IResourceProps[] = [
 export const Resources: React.SFC = () => (
     <>
         <div className="blueprint-resources">
-            {RESOURCES.map(resource => <ResourceCard key={resource.fileName} {...resource} />)}
+            {RESOURCES.map(resource => (
+                <ResourceCard key={resource.fileName} {...resource} />
+            ))}
```

Comment:
This file was failing lint in my PR also. I went ahead and fixed it too.

Reply:
yeah it was fixed on master with #3374